lib/spack/spack/mirror.py

```diff
@@ -69,6 +65,10 @@ def __init__(self, fetch_url, push_url=None,
         self._push_url = push_url
         self._name = name
 
+    def __eq__(self, other):
+        return (self._fetch_url == other._fetch_url and
+                self._push_url == other._push_url)
```

Comment:
This seems orthogonal to the PR in  general

Reply:
Yes, and it has been reverted!